tools/crm_mon.c

```diff
@@ -3923,7 +3923,7 @@ static void crm_diff_update_v2(const char *event, xmlNode * msg)
             free(local_node);
 
         } else {
-            crm_err("Ignoring %s operation for %s %p, %s", op, xpath, match, name);
+            crm_debug("Ignoring %s operation for %s %p, %s", op, xpath, match, name);
```

Comment:
I'm guessing the intent here was to highlight any unsupported diff operations that can't be handled by crm_mon (whether overlooked in development, or due to mixed-version cluster). Is there a specific operation (or small set of operations) that you want to ignore? We could just add an "else if" to ignore those with just a trace (if anything), and maybe make this message more descriptive (something like "Unable to process unrecognized CIB change operation '%s'").